csrankings-j.csv

```diff
@@ -2416,6 +2416,7 @@ Junjun Jiang,Harbin Institute of Technology,http://homepage.hit.edu.cn/jiangjunj
 Junliang Xing,Chinese Academy of Sciences,https://sites.google.com/site/junliangxing,jSwNd3MAAAAJ
 Junlin Lu,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=5937,NOSCHOLARPAGE
 Junlin Zhou,UESTC,http://en.uestc.edu.cn/index.php?m=content&c=index&a=show&catid=79&id=4779,pOmjyCEAAAAJ
+Junmin Xiao,Chinese Academy of Sciences,https://people.ucas.edu.cn/~xiaojunmin,NOSCHOLARPAGE

```

Comment:
Web page does not indicate that this person can solely advise PhD students.

Reply:
I've removed him from this PR.